### DISCUSSION TITLE
¿How Generate Mesh from ply in Swift?

### DISCUSSION THREAD

**Discussion by wikyburns** [cnr-isti-vclab/PyMeshLab#107] [Q&A] [answered]
Hi guys,

I was trying to generate a mesh in Swift actually using trueDepth to get the PointClouds. Actually I'm using pymeshlab server to convert the .ply to .obj or .stl. 

Is there any way to user pymeshlab on swift5? yes, that sounds really crazy. Or anybody knows any third party to swift 5 to "ball pivoting reconstruction" and then export to .obj or .stl?

Thanks!

**Comment by alemuntoni** [comment, marked as answer]
I think you should ask these type of questions in a more generic forum, like stackoverflow.
Here we discuss about pymeshlab, which is a python library that binds meshlab's filters and functionalities in python. It has nothing to do with swift.

**Reply by jinggooo** [reply]
Then, is there any library or function that can be convert ply to obj , scn ?
Or Can render ply in view with swift?

Im trying to show ply with SCNScene , SceneView but i couldn't find a way
